### DISCUSSION TITLE
Need counters in each collection tabular view data count based on categories easily

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#11937] [Archive]
Need counters in each collection tabular view data count based on categories easily

it must be different for each collection? We should be able to configure it in each collection Settings section? Is anyone developing it?

![unknown (8)](https://user-images.githubusercontent.com/36895131/156610307-43bdc272-4676-49c8-9719-8c3ec57d48b7.png)

**Comment by benhaynes** [comment]
No need to spam: https://github.com/directus/directus/discussions/11863
